raycicmd/make.go

```diff
@@ -90,10 +90,15 @@ func makePipeline(repoDir string, config *config, info *buildInfo) (
 
 	c := newConverter(config, info)
 
-	// TODO(aslonnie): use select filter when selects is set.
-	filter, err := newTagsStepFilter(config.SkipTags, config.TagFilterCommand)
-	if err != nil {
-		return nil, fmt.Errorf("run tag filter command: %w", err)
+	var filter *stepFilter
+	if len(info.selects) > 0 {
+		filter = newSelectStepFilter(config.SkipTags, info.selects)
```

Comment:
qq: if the select is skipped, what will be the result?

Reply:
> qq: if the select is skipped, what will be the result?

do you mean skipped by buildkite? rayci tracks dependencies and selects a subgraph based on the dependency. it does not use or care about buildkite condition / skipping.

Reply:
or not exactly sure what you mean by the question..

Reply:
i mean skipped by the config.SkipTags